src/Storages/MergeTree/MergeTreeSettings.h

```diff
@@ -61,6 +61,8 @@ struct Settings;
     M(UInt64, merge_selecting_sleep_ms, 5000, "Sleep time for merge selecting when no part selected, a lower setting will trigger selecting tasks in background_schedule_pool frequently which result in large amount of requests to zookeeper in large-scale clusters", 0) \
     M(UInt64, merge_tree_clear_old_temporary_directories_interval_seconds, 60, "The period of executing the clear old temporary directories operation in background.", 0) \
     M(UInt64, merge_tree_clear_old_parts_interval_seconds, 1, "The period of executing the clear old parts operation in background.", 0) \
+    M(UInt64, merge_tree_clear_old_broken_detached_parts_interval_seconds, 1ULL * 3600 * 24 * 30, "The period of executing the clear old broken detached parts operation in background.", 0) \
```

Comment:
This description makes you think that this operation is done once each 30 days, it's not correct.